mappings/net/minecraft/client/gui/screen/TitleScreen.mapping

```diff
@@ -1,8 +1,30 @@
 CLASS czy net/minecraft/client/gui/screen/TitleScreen
-	FIELD a panoramaCubeMap Ldkg;
-	FIELD b panoramaOverlay Lqs;
+	CLASS czy$a Warning
+		FIELD b line2Width I
+		FIELD c startX I
+		FIELD d startY I
+		FIELD e endX I
+		FIELD f endY I
+		FIELD g line1 Ljm;
+		FIELD h line2 Ljm;
```

Comment:
`firstLine`, `secondLine`

Reply:
Sorry for the late reply, I am currently experiencing an internet outage at home.
The translation strings read `.line1.` and `.line2.`. If first and second is preferred nonetheless, I can change it tomorrow.

Reply:
this should be fine. no worries.